ontology/htn.md

```diff
@@ -12,7 +12,6 @@ license:
   label: CC BY 4.0
   url: http://creativecommons.org/licenses/by/4.0/
 tracker: https://github.com/aellenhicks/htn_owl/issues
-source_url: https://raw.githubusercontent.com/aellenhicks/htn_owl/master/htn.owl
```

Comment:
maybe use `products` instead?

Reply:
No, you are absolutely right, this does not belong here - this goes into the purl config, where it already is.